### DISCUSSION TITLE
Possibility to get complete set of admin levels for Reverse Geocoding

### DISCUSSION THREAD

**Discussion by Zyrix** [osm-search/Nominatim#3270] [Q&A]
I'm trying to extract information about admin levels for several coordinates using reverse geocoding. In order to get consistent information I use format=geocodejson. This works fine and a lot of information is returned including some admin levels. As far as I understand some information is included in the admin dict and some is mapped between admin levels and clearnames. However I'm missing information about admin_level_5 (state district) in an area where it should be available.

API call: `https://nominatim.openstreetmap.org/reverse?lat=48.1225204&lon=11.5592957&format=geocodejson`
I expected 'Upper Bavaria' to be included in the reponse as state_district or admin_level5

**Comment by lonvia** [comment]
Boundaries of admin level 5 are explicitly excluded from the address for Germany, see https://github.com/osm-search/Nominatim/blob/master/settings/address-levels.json#L132. They are usually not relevant for addressing and thus caused more harm than good. You can set up a custom database for Nominatim where you delete the line above and should get the full administrative hierarchy.
